internal/server/static_fs_test.go

```diff
@@ -1,68 +1,77 @@
 package server
 
 import (
+	"net/http"
 	"testing"
+	"testing/fstest"
 
-	"github.com/spf13/afero"
 	"gotest.tools/v3/assert"
 )
 
 func TestStaticFileSystemOpensFile(t *testing.T) {
-	fs := afero.NewHttpFs(afero.NewMemMapFs())
-	_, err := fs.Create("ui/dashboard.html")
-	assert.NilError(t, err)
+	fs := fstest.MapFS{
+		"ui/static/foo.html": {
+			Data: []byte("<html></html>"),
+		},
+	}
 
 	sfs := &StaticFileSystem{
-		base: fs,
+		base: http.FS(fs),
 	}
 
-	f, err := sfs.Open("dashboard.html")
+	f, err := sfs.Open("foo.html")
 	assert.NilError(t, err)
 
 	stat, err := f.Stat()
 	assert.NilError(t, err)
-	assert.Equal(t, stat.Name(), "dashboard.html")
+	assert.Equal(t, stat.Name(), "foo.html")
 }
 
 func TestStaticFileSystemAppendDotHtml(t *testing.T) {
-	fs := afero.NewHttpFs(afero.NewMemMapFs())
-	_, err := fs.Create("ui/dashboard.html")
-	assert.NilError(t, err)
+	fs := fstest.MapFS{
+		"ui/static/foo.html": {
+			Data: []byte("<html></html>"),
+		},
+	}
 
 	sfs := &StaticFileSystem{
-		base: fs,
+		base: http.FS(fs),
 	}
 
-	f, err := sfs.Open("dashboard")
+	f, err := sfs.Open("foo")
 	assert.NilError(t, err)
 
 	stat, err := f.Stat()
 	assert.NilError(t, err)
-	assert.Equal(t, stat.Name(), "dashboard.html")
+	assert.Equal(t, stat.Name(), "foo.html")
 }
 
 func TestStaticFileSystemExists(t *testing.T) {
-	fs := afero.NewHttpFs(afero.NewMemMapFs())
-	_, err := fs.Create("ui/dashboard/foo")
-	assert.NilError(t, err)
+	fs := fstest.MapFS{
+		"ui/static/foo.html": {
+			Data: []byte("<html></html>"),
+		},
+	}
 
 	sfs := &StaticFileSystem{
-		base: fs,
+		base: http.FS(fs),
 	}
 
-	exists := sfs.Exists("/", "/dashboard")
+	exists := sfs.Exists("/", "/foo")
 	assert.Equal(t, exists, true)
 }
 
 func TestStaticFileSystemExistsAppendDotHtml(t *testing.T) {
-	fs := afero.NewHttpFs(afero.NewMemMapFs())
-	_, err := fs.Create("ui/dashboard/foo.html")
-	assert.NilError(t, err)
+	fs := fstest.MapFS{
+		"ui/static/foo.html": {
+			Data: []byte("<html></html>"),
+		},
+	}
 
 	sfs := &StaticFileSystem{
-		base: fs,
+		base: http.FS(fs),
 	}
 
-	exists := sfs.Exists("/", "/dashboard/foo")
+	exists := sfs.Exists("/", "/foo.html")
```

Comment:
Doesn't this change the meaning of the test? The test name suggests we are trying to cover when `/foo` is queried that it looks for a file called `foo.html`.

Does this still work?
```suggestion
	exists := sfs.Exists("/", "/foo")
```

Reply:
I applied this change in a commit, and the test still passes.